server/TracyWorker.cpp

```diff
@@ -2411,6 +2426,19 @@ const char* Worker::GetString( const StringIdx& idx ) const
     return m_data.stringData[idx.Idx()];
 }
 
+uint32_t Worker::AddNewString(const char* newString)
```

Comment:
Why is there a need for a new function to add strings?

Reply:
There is no need really, the only thing diff was this one was public and had that extra assert.
I've removed it and made StoreString public instead.